packages/apps/transfer/src/pages/transfer/module-explorer/networks/[network]/chains/[chain]/modules/[module]/index.tsx

```diff
@@ -0,0 +1,130 @@
+import { ChainwebChainId } from '@kadena/chainweb-node-client';
+import { StringContractDefinition } from '@kadena/pactjs-generator';
+
+import { kadenaConstants, Network } from '@/constants/kadena';
+import { convertIntToChainId } from '@/services/utils/utils';
+import dynamic from 'next/dynamic';
+import { useRouter } from 'next/router';
+
+const AceViewer = dynamic(import('@/components/Global/Ace'), {
+  ssr: false,
+});
+
+import { Container, Details, EditorGrid, StyledListItem } from './styles';
+
+import { describeModule } from '@/services/modules/describe-module';
+import Debug from 'debug';
+import useTranslation from 'next-translate/useTranslation';
+import React, { FC, useEffect, useState } from 'react';
+
+const ModulePage: FC = () => {
+  Debug('kadena-transfer:pages:transfer:module-explorer:module');
```

Comment:
Not sure where this is how `debug` should be used? Especially not in a React component on each render.